ember-scoped-css/src/lib/scoped-css-preprocessor.js

```diff
@@ -67,12 +71,57 @@ class ScopedFilter extends Filter {
       return '';
     }
   }
+  async postProcess(results, relativePath) {
+    if (relativePath.endsWith('.module.css')) {
```

Comment:
this should *probably* be a config option, but I think since usage is low, we can skip it

Reply:
I'll add a config option to opt in

Reply:
Misunderstood, will look at doing that in a later PR since `module.css` is also considered elsewhere.

However, I do think I'd like to check for CI to skip the whole thing since CI doesn't do a second pass and we'd be doing extra work for nothing.

```js
   if (process.env.CI || relativePath.endsWith('.module.css')) {
      return results;
    }
```